polygeist/tools/cgeist/Lib/CGDecl.cc

```diff
@@ -120,7 +120,7 @@ ValueCategory MLIRScanner::VisitVarDecl(clang::VarDecl *Decl) {
     if (isa<LLVM::LLVMPointerType>(Glob.getTypes().getMLIRType(
             Glob.getCGM().getContext().getPointerType(Decl->getType())))) {
       auto GSF = Glob.getOrCreateLLVMGlobal(
-          Decl, (Function.getName() + "@static@").str());
+          Decl, (Function.getName() + "@static@").str(), FunctionContext::Host);
```

Comment:
Is it intended to always use `Host` here?

Reply:
Yup, to match line 131.